### ISSUE TITLE
ERROR in ./README.md

### ISSUE THREAD

**Issue by zuolizhu** [vuejs/vuepress#50]
I just installed vuepress and tried to run `vuepress dev`, 
```
ERROR in ./README.md
Module build failed: TypeError: Cannot read property 'join' of undefined
    at Object.module.exports (C:\Users\xxy\AppData\Roaming\npm\node_modules\vuepress\lib\webpack\markdownLoader.js:81:17)
 @ C:/Users/xxy/AppData/Roaming/npm/node_modules/vuepress/lib/app/.temp/routes.js 10:8-68
 @ C:/Users/xxy/AppData/Roaming/npm/node_modules/vuepress/lib/app/app.js
 @ C:/Users/xxy/AppData/Roaming/npm/node_modules/vuepress/lib/app/clientEntry.js
 @ multi C:/Users/xxy/AppData/Roaming/npm/node_modules/vuepress/lib/app/clientEntry.js
```
Thanks!

**Comment by zuolizhu** [comment]
I do created a .md file before I run `vuepress dev`.

**Comment by mapl** [comment]
watch out for any leading and tailing double quotes

**Comment by yyx990803** [comment]
Should be fixed by ed33515 - also updated getting started instructions...
